valhalla/midgard/tiles.h

```diff
@@ -123,7 +123,7 @@ template <class coord_t> class Tiles {
    * @param   y   y coordinate
    * @return  Returns the tile row. Returns -1 if outside the tile system bounds.
    */
-  int32_t Row(const float y) const {
+  int32_t Row(const double y) const {
```

Comment:
to correctly adhere to the template parameter we should probably use:

```suggestion
  int32_t Row(const typename coord_t::value_type y) const {
```

Reply:
agree, fixed.